### PR TITLE
Drag tabs to resort them #12

### PR DESCRIPTION
**Pull request by Perkovec**

### CODE REVIEW

File: src/utils/dnd-light.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      'drag-and-drop',
+      'drag-start',
+      'drag',
+      'drag-over',
+      'drag-enter',
+      'drag-leave',
+      'drag-end',
+      'drop'
+    ],
+    bind: function () {
```
Comment (egoist) [inline src/utils/dnd-light.js]: this file should be updated to ES6

**Comment by egoist** [comment]
Maybe we can try to highlight to tab we drag to, like this:

test

**Comment by Perkovec** [comment]
@egoist nice

**Comment by Perkovec** [comment]
@egoist there are no more ideas how to update it to ES6

File: src/utils/dnd-light.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
-      'drag-start',
-      'drag',
-      'drag-over',
-      'drag-enter',
-      'drag-leave',
-      'drag-end',
-      'drop'
-    ],
+    params: [...params],
     bind: function () {
```
Comment (egoist) [inline src/utils/dnd-light.js]: and the function too 😅

**Comment by Perkovec** [comment]
@egoist lost context with arrow function

**Comment by egoist** [comment]
`bind() {}` would not lost context

**Comment by Perkovec** [comment]
@egoist thanks

**Comment by egoist** [comment]
Merged.

**Pull request closed without merging**